<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundCustomConf</class>
 <widget class="QWidget" name="BackgroundCustomConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BackgroundCustomConf</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="colorGradientConf" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="colorGradientChoice" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="colorChoice">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Background Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="gradientChoice">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Background Gradient</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="chosenCustomBackground" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="chosenCustomBackgroundLayout"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="customWidthHeightArea" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="customWidthArea" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelCustomWidth">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Width :  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="customWidthChoice">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelPxWidth">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>px</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="customHeightArea" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelCustomHeight">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Height :  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="customHeightChoice">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelPxHeight">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>px</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelWarningWH">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Small values of Width and Height are not recommended.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
